test/neo4clj/client_test.clj

```diff
@@ -0,0 +1,207 @@
+(ns neo4clj.client-test
+  (:require [clojure.test :as t]
+            [java-time :as time]
+            [neo4clj.client :as sut]
+            [neo4clj.test-utils :as test-utils]))
+
+(t/deftest with-session
+  "This test also tests the create-session and execute! function"
```

Comment:
`deftest` doesn't take a doc-string.

This should probably just be a code comment (`;;`) or somehow integrated into the text in `t/testing`.

Remember `t/testing` can be nested:
```clojure
(t/testing "Testing this thing"
  (t/testing "with A"
    ...)
  (t/testing "without A"
    ...))
```